homeassistant/components/statistics/services.yaml

```diff
@@ -1,3 +1,39 @@
 reload:
   name: Reload
   description: Reload all statistics entities.
+
+reset_buffer:
+  name: Reset buffer
+  description: >-
+    Remove stored state values from a statistics sensor.
+    A number of most recent state values can be kept in memory.
+    By default, all values will be forgotten, which leads to a temporarily unknown state for most statistical characteristics.
+    In case both "keep count" and "keep age" are provided, values which meet one of the two conditions are kept.
+  target:
+    entity:
+      # integration: statistics
+      domain: sensor
```

Comment:
@emontnemery the commented out variant didn't work. Any pointers as to why?

Reply:
I'm not sure, but we want the integration filter in the selector.
There are other services which limit to sensors from a certain integration, for example homeassistant/components/advantage_air/services.yaml, so it should be possible to make it work.

Reply:
Yes, I also found other integrations (like Sonos) which use this method. In my case the Home Assistant Service UI does not offer any entities to select from.

Reply:
This is due to a frontend bug / limitation where filtering by integration does not work for entities not tied to a config entry (basically all yaml-configured entities).
The groundwork for supporting filtering by integration for all entities was done here: https://github.com/home-assistant/frontend/pull/12149, but that needs to be extended to the target selector too.

Long story short, we need to remove the `integration: statistics` filter for now.